components/PostArticle.vue

```diff
@@ -84,6 +85,11 @@ export default {
       type: String,
       required: true
     }
+    // applyLink: {
```

Comment:
Not sure what this does, we should explain it or delete it

Reply:
remvoed